cocos/editor-support/spine/SkeletonAnimation.h

```diff
@@ -34,7 +34,7 @@
 
 #include <spine/spine.h>
 #include <spine/SkeletonRenderer.h>
-#include "cocos2d.h"
```

Comment:
OK, I'll revert the changes to spine here. and try to make a PR at https://github.com/EsotericSoftware/spine-runtimes.